aireview/src/aireview/main.py

```diff
@@ -1,23 +1,23 @@
 import asyncio
+import logging
 
 from aireview.application.controllers.review_controller import ReviewController
 from aireview.config import Config
 from aireview.domain.services.code_analyzer import CodeAnalyzer
 from aireview.infrastructure.dust_client import DustClient
 from aireview.infrastructure.github_client import GitHubClient
 
+logger = logging.getLogger(__name__)
 
-
-def main():
+def main(pull_request: int):
     config = Config()
```

Comment:
**style**: Logger is initialized but not used. Consider adding logging statements for key operations or remove if unused